File: sdks/python/types/transcription_response.py
import typing

import pydantic
from ..core.pydantic_utilities import IS_PYDANTIC_V2, UniversalBaseModel


class TranscriptionResponse(UniversalBaseModel):
    """
    `json` returns `{ text }`. `verbose_json` adds `task`, `language`, `duration`, and `segments` and/or `words`.
    """

    text: str

    if IS_PYDANTIC_V2:
        model_config: typing.ClassVar[pydantic.ConfigDict] = pydantic.ConfigDict(extra="allow", frozen=True)  # type: ignore # Pydantic v2
    else:

        class Config:
            frozen = True
            smart_union = True
            extra = pydantic.Extra.allow

File: sdks/typescript/api/types/TranscriptionResponse.ts
/**
 * `json` returns `{ text }`. `verbose_json` adds `task`, `language`, `duration`, and `segments` and/or `words`.
 */
export interface TranscriptionResponse {
    /** The transcript. */
    text: string;
    /** Accepts any additional properties */
    [key: string]: any;
}
